src/mempool/mempool.service.ts

```diff
@@ -0,0 +1,54 @@
+import { Injectable } from "@nestjs/common";
+import { Cron, CronExpression } from "@nestjs/schedule";
+import { getRPCProvider } from "../utils/client";
+import { TransactionIdentifier } from "../models/TransactionIdentifer";
+import { OperationFactory } from "../models/OperationFactory";
+import { Transaction } from "../models/Transaction";
+
+@Injectable()
+export class MempoolService {
+  
+  private provider = getRPCProvider();
+  private mempool:TransactionIdentifier[] = [];  
```

Comment:
I suggest replacing this with Map<string, TransactionIdentifier> where key is the tx.hash. This will allows the best way of finding, adding and replacing transactions.

Reply:
Good suggestion, Thx.